.goreleaser.yml

```diff
@@ -0,0 +1,36 @@
+version: 2
```

Comment:
Why this file is on the root path and hidden?

Reply:
this is the convention for goreleaser:
https://goreleaser.com/how-it-works/